Angeline Polidano/week_1/calculator/js/main.exercises.js

```diff
@@ -0,0 +1,44 @@
+// # The Calculator
+
+// ## Part 1
+// - Write a function called squareNumber that will take one argument (a number), square that number, and return the result. It should also log a string like "The result of squaring the number 3 is 9."
+
+//return results
+
+const squareNumber = function (numbera, numberb) {
```

Comment:
Minor nitpick, 

numbera  ->  numbersA

You've used [camel case ](https://www.computerhope.com/jargon/u/uppercase.png)correctly elsewhere, could be used here as well